src/System.Memory/src/System/MemoryExtensions.Fast.cs

```diff
@@ -8,9 +8,9 @@
 namespace System
 {
     /// <summary>
-    /// Extension methods for Span&lt;T&gt;.
+    /// Extension methods for Span&lt;T&gt;, Memory&lt;T&gt;, and friends.
```

Comment:
```C#
/// Extension methods for Span{T}, Memory{T}, and friends.
```

Reply:
Question: Why is {T} preferred over \<T\>?

Should I also update it in other places like
`IEquatable&lt;T&gt;.Equals(T)` (https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/System/SpanExtensions.cs#L16)?

What about in the exception message string:
`Memory&lt;T&gt; has been disposed.` (https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/Resources/Strings.resx#L136)?

I see the use xml escape characters elsewhere too:
https://github.com/dotnet/corefx/blob/master/src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.Builder.cs#L265
https://github.com/dotnet/coreclr/blob/master/tests/src/CoreMangLib/cti/system/collections/generic/dictionary/dictionaryicollectionadd.cs#L8

Reply:
> Why is `{T}` preferred over `<T>`?

You can't write `<T>` in XML comments because it's interpreted as XML.  And if you write `&lt;T&gt;`, that's just text like any other.  If you write `{T}`, that's parsed by the C# compiler as a generic type parameter.

Reply:
> I see the use xml escape characters elsewhere too:

And that's "incorrect", too.

> What about in the exception message string

That should be `Memory<T>`.

Reply:
> that's parsed by the C# compiler as a generic type parameter.

The compiler doesn't ignore comments during parsing?

> What about in the exception message string
That should be `Memory<T>`.

Can you clarify this? Should I leave it as `Memory&lt;T&gt;` and it will show up as `Memory<T>` as expected? I can't change it to because it is interpreted as XML.
`<value>Memory<T> has been disposed.</value>`

Should I change it to the following?
`<value>Memory{T} has been disposed.</value>`

Reply:
> The compiler doesn't ignore comments during parsing?

XML comments are special.

> Can you clarify this?

Ah, I misunderstood the issue.  Leaving it as `Memory&lt;T&gt;` should be fine.